Status.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Http;
+
+class Status
+{
+    public const OK = 200;
+    public const ERROR = 500;
+    public const UPDATED = 201;
+}
```

Comment:
you dont need to create enum class. you can use `Symfony\Component\HttpFoundation\Response`
`Response::HTTP_OK` will return 200